Diplomas/DiplomaSetup.php

```
@@ -29,7 +29,11 @@ function pl_diploma_ensure_tables() {
 		) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4");
 	} else {
 		// Add title enable column if upgrading from earlier version
-		safe_w_sql("ALTER TABLE PLDiplomaConfig ADD COLUMN IF NOT EXISTS PlDcTitlesEnabled TINYINT(1) NOT NULL DEFAULT 0");
+		$Rc = safe_r_sql("SHOW COLUMNS FROM PLDiplomaConfig LIKE 'PlDcTitlesEnabled'");
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $Rc. Configured minimum length is 3. (undefined)

(ShortVariable)